bin/Powheg/run_pwg.py

```diff
@@ -329,6 +329,10 @@ def runGetSource(parstage, xgrid, folderName, powInputName, process, tagName) :
 if [[ -s ./JHUGen.input ]]; then
   jhugen=$(expr $jhugen + 1)
   echo "JHUGen activated!"
+  #for decay weights in H->WW and H->ZZ
```

Comment:
naive question: so far we produced already HZZ and HWW gridpacks, why is this addition needed?